contracts/Comptroller.sol

```diff
@@ -752,6 +852,14 @@ contract Comptroller is ComptrollerStorage, ComptrollerInterface, ComptrollerErr
                 return (Error.MATH_ERROR, 0, 0);
             }
 
+            /// @dev VAI Integration^
+            if (vars.sumCollateral > vars.mintedVAIMantissa) {
```

Comment:
what if the collateral and mintedVAI values are equal? should it still throw the MATH_ERROR exception?